config/locales/en.helpers.yml

```diff
@@ -69,19 +69,31 @@ en:
     have_fun: "Good luck with your training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "your match (%{team_home} vs. %{team_away}) in the event %{name}"
+    start_time: "When? On %{start_date} at %{start_time} Uhr"
```

Comment:
I guess you meant something like “at %{start_time} o'clock” or simply “at %{start_time}” instead of “at %{start_time} Uhr”? 😉

Reply:
For sure!